osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -479,12 +477,18 @@ private void revert()
 
         protected void Cut()
         {
+            if (!canCut.Value)
+                return;
```

Comment:
What are these changes doing here?

Reply:
They are a bit unrelated, but not wrong?

Reply:
They may not be wrong but the changes are out of scope of this pull. The relevant menu items already check the `can{Cut,Copy,Paste}` bindables to determine if the menu items should be disabled. Move these changes to a separate pull, with test coverage, if the lack of these checks in this concrete place causes an actual issue.